There is a discrepancy between where btor2tools installs its header and
where boolector is looking for it.

boolector can download and build btor2tools for us, but we don't want
build systems to be downloading things for us, and we want to control
what flags are used for the build.
--- cmake/FindBtor2Tools.cmake.orig	2021-05-27 19:47:11.000000000 -0500
+++ cmake/FindBtor2Tools.cmake	2022-01-06 07:01:05.000000000 -0600
@@ -11,7 +11,7 @@
 # Btor2Tools_INCLUDE_DIR - the Btor2Tools include directory
 # Btor2Tools_LIBRARIES - Libraries needed to use Btor2Tools
 
-find_path(Btor2Tools_INCLUDE_DIR NAMES btor2parser/btor2parser.h)
+find_path(Btor2Tools_INCLUDE_DIR NAMES btor2parser.h)
 find_library(Btor2Tools_LIBRARIES NAMES btor2parser)
 
 include(FindPackageHandleStandardArgs)
--- src/btormcmain.c.orig	2021-05-27 19:47:11.000000000 -0500
+++ src/btormcmain.c	2022-01-06 07:00:38.000000000 -0600
@@ -9,7 +9,7 @@
 #include "boolectormc.h"
 #include "btormc.h"
 
-#include "btor2parser/btor2parser.h"
+#include <btor2parser.h>
 #include "utils/btorhashint.h"
 #include "utils/btormem.h"
 #include "utils/btoroptparse.h"
--- src/parser/btorbtor2.c.orig	2021-05-27 19:47:11.000000000 -0500
+++ src/parser/btorbtor2.c	2022-01-06 07:00:56.000000000 -0600
@@ -6,7 +6,7 @@
  *  See COPYING for more information on using this software.
  */
 
-#include "btor2parser/btor2parser.h"
+#include <btor2parser.h>
 #include "btormsg.h"
 #include "btorparse.h"
 #include "btortypes.h"
